modules/apps/portal-crypto-hash-generator/portal-crypto-hash-generator-message-digest/src/main/java/com/liferay/portal/crypto/hash/generator/message/digest/factory/MessageDigestHashGeneratorFactory.java

```diff
@@ -0,0 +1,50 @@
+/**
```

Comment:
When introducing a new class that implements an interface
        take into account:

* The class name should end with the interface name
* The package of the class matches the package of the interface



Refer to documentation: